<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TMZ-DPC-2185" failures="1" tests="6" name="Default test" time="152.285" errors="0" timestamp="2023-11-27T16:12:47 IST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="32.305"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="12.012"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.293"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="39.275"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="35.957"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="14.482">
    <failure type="java.lang.AssertionError" message="expected [Recovery mail has been sent to your Email] but found []">
      <![CDATA[java.lang.AssertionError: expected [Recovery mail has been sent to your Email] but found []
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at Definitions.LoginDef.user_clicks_to_send_link_for_reset(LoginDef.java:109)
at âœ½.User clicks to send link for reset(file:///C:/Users/ashekharangya/Git%20Lenskart/TestDreamers_Lenskart/src/test/resources/Feature/Login.feature:31)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
